code/modules/reagents/reagents/food.dm

```diff
@@ -84,6 +84,7 @@
 	taste_multi = 1.5 // stop sugar drowning out other flavours
 	nutriment_factor = 10
 	taste_description = "sweetness"
+	var/blood_gain = 0.2
```

Comment:
besides this change being questionable in itself,
this doesn't even work.. the var existing does nothing as blood_gain handling is a ../reagent/consumable/nutriment/.. thing whereas sugar is not a type of that (which is why you had to put the var/ before blood_gain so it compiled, and should also be getting a "var not used" warning